sdk/src/main/kotlin/io/rover/sdk/ui/blocks/poll/image/ImagePollViewModel.kt

```diff
@@ -137,9 +142,10 @@ internal interface ImagePollViewModelInterface : BindableViewModel, Measurable {
     fun informImagePollOptionDimensions(
         measuredSize: MeasuredSize
     )
-
+    
     fun castVote(selectedOption: String, optionIds: List<String>)
     fun checkForUpdate(pollId: String, optionIds: List<String>)
```

Comment:
a hangover from the old regime, maybe add a comment explaining that you're dead-reckoning the state in the interactor's state machine here in order to make your existing animation implementation work?